modules/ROOT/pages/pages-development.adoc

```diff
@@ -18,8 +18,8 @@ A page resource has the general xref:resource-management.adoc[resource definitio
 If it contains an `Index.groovy` file, this must implement the PageController interface optionally with libraries.
 If you create a custom page with Bonita UI Designer, its required structure and content are right, by default.
```

Comment:
If you create a custom page with Bonita UI Designer, its required structure and content are right, by default.=> If you create a custom page with Bonita UI Designer, its structure and content will be congruent by default.

Reply:
Added